scripts/libsimple_crontab

```diff
@@ -8,21 +8,56 @@ PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
 HOME=/var/www/circulation
 
 # m h dom mon dow user command
+
+# These scripts update internal caches.
+#
+*/5 * * * * root core/bin/run cache_opds_blocks >> /var/log/cron.log 2>&1
+0 */6 * * * root core/bin/run refresh_materialized_views >> /var/log/cron.log 2>&1
+0 4 * * * root core/bin/run update_random_order >> /var/log/cron.log 2>&1
+0 1 * * * root core/bin/run loan_reaper >> /var/log/cron.log 2>&1
```

Comment:
This script removes loans and holds that are super old, to avoid storing loan information for patrons who have stopped using the app.